wiki/en/Running-a-Server.md

```diff
@@ -50,55 +49,47 @@ Unless you plan on hosting more than about 5 players on a slower-speed home conn
 
 - Running a server may require you to adjust any firewalls running on or outside of your machine or cloud host.
 
-- Running a Private Server at home will require you to [port forward](#running-a-private-server) on your router. When running a Public Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
+- Running an Uregistered Server at home will require you to [port forward](#running-an-unregistered-server) on your router. When running a Registered Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
```

Comment:
Ah, I see the port forwarding "problem" is mentioned here already :) .

Reply:
Yes, a relatively recent addition. These docs haven't been changed much otherwise in almost two years I think.